src/model/mmkv.ts

```diff
@@ -11,14 +11,25 @@ export const STORAGE_IDS = {
   NOTIFICATIONS: 'NOTIFICATIONS',
   RAINBOW_TOKEN_LIST: 'LEAN_RAINBOW_TOKEN_LIST',
   SHOWN_SWAP_RESET_WARNING: 'SHOWN_SWAP_RESET_WARNING',
+  GLOBAL: 'global',
+  KEYCHAIN: 'rainbowKeychainLocalStorage',
```

Comment:
I would say we ~should keep this separate

Reply:
So for the global we used to clear that because it was stored in AsyncStorage before. I could agree with the one for keychain though we could remove it from clear actions. WDYT?

Reply:
yeah global makes sense, I think we should keep the keychain key completely separate from this list just to keep it contained

Reply:
Yeah I think I agree. I'd like to have the "nuke" functionality be opt-in instead of opt-out via `STORAGES_THAT_CANT_BE_CLEARED`. Would hate to see us get into a weird broken state if something gets cleared that shouldn't have been cleared.